.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: ruby
+before_install:
+  - gem install bundler
```

Comment:
I'm not actually sure you need the ```gem install bundler``` as I think they handle that if you have a Gemfile. Are you targeting any specific versions of Ruby?

I normally add:

```
rvm:
  - 2.2.2
cache: bundler
```

too for example.